src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  font-size: 13px;
+  font-family: Arial;
+}
+
+p {
+  margin: 0;
+}
+
+ol {
+  margin: 0;
+}
+
+h1 {
+  font-weight: bold;
+  font-size: 17px;
+  margin: 0;
+}
+
+h2 {
+  font-weight: normal;
+  font-size: 15px;
+  padding: 1em 0;
+  margin: 0;
```

Comment:
it's not padding, it's  more like margin 1 em
padding is for inner indentations but you need indentations for your container from other containers